components/Navigation/components/NavItem/NavItem.tsx

```diff
@@ -23,9 +23,9 @@ export const NavItem: FC<INavItemProps> = ({ active, icon, badge, ...props }) =>
 
   return (
     <div {...props} className={rootClassName} onMouseEnter={toggleHover} onMouseLeave={toggleHover}>
-      {badge ? <Badge color={active || isHovered ? 'outline-blue' : 'blue'}>{badge}</Badge> : null}
+      {badge ? <Badge>{badge}</Badge> : null}
       <a className={iconClassName}>
-        <Icon color={themeColors.icon} {...((active || isHovered) && { color: 'white' })} />
+        <Icon color={themeColors.icon} {...((active || isHovered) && { color: 'var(--blue)' })} />
```

Comment:
please use UPPER_BLUE_COLOR
```suggestion
        <Icon color={active || isHovered ? UPPER_BLUE_COLOR : themeColors.icon} />
```

Reply:
Done